### PR TITLE
feat: split justificatif into /documents and /entreprise and /annonces

### LINKED ISSUE #828: [RNE] Fallback des appels justificatifs, dirigeants et RBE sur les données de l'API de recherche
Cette issue a besoin d'avoir les informations de base d'immatriculation dans l'API.

Mettre en place un fallback la route `/data-fetching/rne` vers les dirigeants et l'immatriculation de l’API de recherche va permetre de : 
- se passer du scrapping de data.inpi
- s'autoriser a diminuer drastiquement le timeout

### LINKED ISSUE #975: Supprimer la page justificatif
Supprimer la page justificatif

## Migrer les justificatifs d'existence 
Dans la page documents

## Migrer les informations d'immatriculation 
A basculer dans l'onglet résumé pour les entreprises

## Revoir la définition de l'etat administratif
actif/fermé dans la balise title pour les entreprises

### PR DESCRIPTION
**Pull request by XavierJp**
Todo : 
- [x] ~raccourcis vers documents~
- [x] Inscrite au RNA
- [x] Inscrite a XXX , le ...
- [x] observations ajouter un `.`
- [x] resumé dans annonces et observations
- [x] redirection de `justificatif/<siren>` vers `documents/<siren>`
- [x] Parser les observations sur data.inpi;fr + add metadata warning
- [x] justif dans la page
- [x] Proxy refactoring is needed : 
	- a route for RNE call
		- returns dirigeants,  observations and immatriculation
	- a route for RNE fallback
		- returns immatriculation, observations (no need for dirigeant ?)
- [x] changelog
- [x] fallback /dirigeants to API recherche
- [x] cleaning
- [x] handle immatriculation error (404 or 5xx from INPI / from recherche)
   - if recherche down or dateInpi = null -> call Proxy
   - proxy must return a clone of immatriculation object
   - if proxy down we get a IAPINotResponding
- [x] tests

closes #975 
closes #828

### CODE REVIEW

**Comment by XavierJp** [comment]
@johangirod @HAEKADI sorry for this MASSIVE overhaul. This is not yet finished and lacks some tests. Yet it is ready for review : 
- delete `/justificatif` page
- move immatriculation data to `Fiche résumé`
image
- move justificatif PDF to `/documents`
- move observations to `/annonces`
- revamp the main section 
image
- update changelog

File: app/(header-default)/entreprise/[slug]/_components/inscriptions.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        <Icon slug="download">
+          <a href={`/documents/${uniteLegale.siren}`}>
+            Annonce de création au JOAFE
+          </a>
+        </Icon>
+      }
+    >
+      <InformationTooltip
+        tabIndex={undefined}
+        label="Cette structure est inscrite au Registre National des Association (RNA)."
```
Comment (HAEKADI) [inline app/(header-default)/entreprise/[slug]/_components/inscriptions.tsx:189]:
```suggestion
        label="Cette structure est inscrite au Registre National des Associations (RNA)."
```

File: data/faq/registre-des-beneficiaires-effectifs.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ body: |
   - nom, prénoms, pseudonyme et nom d'usage
   - mois et année de naissance
   - pays de résidence et nationalité
-  - nature et modalité de controle détenus par le bénénficiaire
+  - nature et modalité de controle détenus par le bénéficiaire
```
Comment (HAEKADI) [inline data/faq/registre-des-beneficiaires-effectifs.yml:32]:
```suggestion
  - nature et modalité de contrôle détenus par le bénéficiaire
```

**Review by HAEKADI — APPROVED** [review]
Very cool 🚀

**Comment by XavierJp** [comment]
@HAEKADI it is ready. As soon as the tests are green, It is all yours to merge. 
I believe only you can merge it as you poured so much efforts into this. And once again, amazing work. Congratulations and thanks for all the fish 🐬

**Comment by HAEKADI** [comment]
LET'S GOOO

**Pull request merged**